### ISSUE TITLE
Questions about the number of microRNAs

### ISSUE THREAD

**Issue by YangXinyan** [multitalk/miRTalk#1]
To developers

miRTalk is an interesting work, but I have some doubts. What do you think of the fact that it is almost difficult to detect micro RNA in the 10X data. According to 10X genomic, almost no mature micro RNA can be detected in the data (Ref 1-2). As in the Demo data, only 60+ micro RNAs were detected in the end.

Ref 1-2:
https://kb.10xgenomics.com/hc/en-us/articles/360002004271-Is-it-possible-to-capture-microRNAs-in-the-Gene-Expression-assays-
https://kb.10xgenomics.com/hc/en-us/articles/217261666-What-types-of-RNA-can-be-detected-

with regards,
Xinyan Yang

**Comment by multitalk** [comment]
For 10X Genomics data, the miRNAs refer to the pri-miRNA transcripts. For the STRS data(https://doi.org/10.1038/s41587-022-01517-6), the miRNAs  refer to the mature miRNA transcripts.

**Comment by yingnanyou** [comment]
Hi, I encountered an error while using the miRTalk package and would appreciate your help.
1.I ran the following code:
obj <- find_miRNA(object = obj, mir_info = mir_info, mir2tar = mir2tar)
2.Expected result: The function should identify miRNAs in the single-cell data.
3.Actual result: The following error occurred:
Error in find_miRNA(object = obj, mir_info = mir_info, mir2tar = mir2tar) : 
  No miRNAs found in sc_data!
R Version: [R 4.3.0]
